setup.cfg

```diff
@@ -68,5 +68,9 @@ filterwarnings =
     ignore::DeprecationWarning
     ignore::PendingDeprecationWarning
 
+[coverage:run]
+parallel = true
+omit = pipenv/vendor*, pipenv/patched*
+
```

Comment:
Omitting `vendor` and `patched` might hide unused patched and vendor packages.
Do you have a solution on how to avoid this?

Reply:
I forget why I had originally excluded those, but thinking about it now I don't see a strong reason to.  It might have been just because I excluded them from the linting that I copied that same pattern to exclude them from coverage, but I don't think there is a good reason to like there was for the linting.